cves/CVE-2015-3183.yml

```diff
@@ -66,17 +68,17 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 29779fd08c18b18efc5e640d74cbe297c7ec007e
-     note:
-   - commit: c3f98ce69b9c8ddc936380e9d7cf03feca809558
-     note:
-   - commit: 87fcef685c19652353c2e26e90ed1a2aa8099629
-     note:
+  - commit: 29779fd08c18b18efc5e640d74cbe297c7ec007e
```

Comment:
In a quick search through the git commits, I couldn't locate this hash outside of being mentioned in another issue on github. Maybe double check this commit hash to make sure this is right.

Reply:
This is one of multiple hashes I found that for some reason don't show up when you search for them, but if you manually change the URL it shows up: https://github.com/apache/httpd/commit/29779fd08c18b18efc5e640d74cbe297c7ec007e